core/src/mindustry/game/Team.java

```diff
@@ -64,26 +64,18 @@ protected Team(int id, String name, Color color){
         all[id] = this;
 
         palette = new Color[3];
-        palette[0] = color;
+        palette[0] = color.cpy();
         palette[1] = color.cpy().mul(0.75f);
         palette[2] = color.cpy().mul(0.5f);
```

Comment:
this is not need now.

Reply:
Initial Color instances are still required to be able to use Color.set().